am-lib/src/main/java/uk/gov/hmcts/reform/amlib/internal/repositories/AccessManagementRepository.java

```diff
@@ -115,4 +124,12 @@
         + "and (:relationship is null or access_management.relationship = :relationship) "
         + "and (access_management.attribute = :attributeAsString or access_management.attribute like concat(:attributeAsString, '/', '%'))")
     void revokeAccessManagementForAudit(@BindBean ExplicitAccessMetadata explicitAccessMetadata);
+
+    @SqlQuery("select resource_name as caseTypeId, role_name as role, permissions from default_permissions_for_roles "
+        + "where resource_type = 'case' and resource_name in (<caseTypeIds>) "
+        + "and attribute = '' order by resource_name, role_name")
+    @RegisterBeanMapper(RolePermissionsForCaseTypeEnvelope.class)
+    @RegisterBeanMapper(DefaultRolePermissions.class)
```

Comment:
Please move the queries to the defaultroleImportService repository

Reply:
done